arctic/store/version_store.py

```diff
@@ -162,7 +162,7 @@ def list_symbols(self, all_symbols=False, snapshot=None, regex=None, **kwargs):
                                       }
                          }])
 
-        results = self._versions.aggregate(pipeline)
+        results = self._versions.aggregate(pipeline, allowDiskUse=True)
```

Comment:
Tbh I think the external sort is the real issue here - and we shouldn't be doing it for efficiency reasons.  Can we fix the query (or index) so that we don't require a full sort?

Reply:
I don't think any sorting is happening here. Just that if memory spills, it will be able to write to disk.
This is a new feature in v2.6 while v2.4 is used in circleci. (And no such problem in travis-ci)

Reply:
It's spilling the temporary results to disk because the results of the aggregation pipeline don't fit in memory.  This is quite expensive (compared with an indexed read). 

Can the code / indexes be modified instead to not require external disk?

Reply:
If the size of the returned list of symbols exceeds 16MB cap, spilling is inevitable when using aggregate.